app/models/user.rb

```diff
@@ -1,2 +1,17 @@
 class User < ApplicationRecord
+  before_save { self.email = email.downcase }
```

Comment:
Write to a private method and call it to before_save
![image](https://user-images.githubusercontent.com/65896888/90614864-6e4e7100-e235-11ea-8903-50edb80db7bd.png)